lib/db.js

```diff
@@ -512,19 +514,30 @@ module.exports = (
       lastAccessTime: token.lastAccessTime,
       createdAt: token.createdAt,
       isMemoryToken: true
-    }]
+    }
     let sessionTokens = []
+    return getGeoData(ip)
```

Comment:
> as I'm the only one who think so

Not true, I liked the idea :) , but the logistics of storing the ipHash are just too much :(